examples/cxx/publish_subscribe_dynamic_data/src/publisher.cpp

```diff
@@ -38,19 +38,19 @@ auto main() -> int {
                          .create()
                          .expect("successful publisher creation");
 
-    auto counter = 1;
+    auto counter = 0;
 
     while (node.wait(CYCLE_TIME).has_value()) {
-        counter += 1;
-
-        auto required_memory_size = (8 + counter) % 16; // NOLINT
+        auto required_memory_size = (counter % 16) + 1; // NOLINT
```

Comment:
```suggestion
        const auto required_memory_size = (counter % 16) + 1;
```
The same can be done on line 35

Reply:
@elBoberido Fixed